### ISSUE TITLE
CTM Overlays do not work on non-standard blocks

### ISSUE THREAD

**Issue by realjackasterisk** [sp614x/optifine#1270]
(reposting from Discord server)
I'm having a small issue with CTM. I'm trying to use the `overlay_fixed` method on flowers to give them a grass tint, but they won't change at all. It works for normal blocks perfectly fine.

I'm using standalone, preview D1 on 1.12.2.

Test resource pack: https://drive.google.com/file/d/1hQrwjP-COHisxnAFQmueYzJMTMKias3v/view?usp=sharing
This should add a grey blob to the sides of grass blocks and dandelions, and then apply the grass tint color to it. Both .properties files are essentially the same.

**Comment by ignamiranda** [comment]
Yes, please! I'm trying the exact same thing to make sapling leaves match the tint of leaf blocks with no luck

**Comment by Luke18033** [comment]
Seems to be a duplicate of #1068?
